src/blocks/mrc_call_python_function.ts

```diff
@@ -37,6 +37,7 @@ const FUNCTION_KIND_MODULE = 'module';
 const FUNCTION_KIND_STATIC = 'static';
 const FUNCTION_KIND_CONSTRUCTOR = 'constructor';
 const FUNCTION_KIND_INSTANCE = 'instance';
+const FUNCTION_KIND_INSTANCE_WITHIN = 'instance_within';
```

Comment:
should these be string enums so the compiler will only allow the ones we want?

Reply:
Yes!